nextcord/application_command.py

```diff
@@ -593,7 +593,7 @@ def __init__(self, callback: Optional[Callable] = None, parent_cog: Optional[Cli
 
         Parameters
         ----------
-        callback: Optional[Callable]
+        callback: Optional[:class:`Callable`]
```

Comment:
Nearly every other reference to typing.* doesn't use links so the types in the docstrings are shorter.